.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
`I'm always frustrated` -> `It is frustrating`
Try to avoid personalization to circumvent future "Well, it makes me frustrated so that's my argument", whereas "It is frustrating" can allow for more discussion as to why and explanation with reasoning.